scripts/background.js

```diff
@@ -3,11 +3,11 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (
     changeInfo.status === 'complete' &&
     tabUrl &&
-    tabUrl.includes('linkedin.com/feed')
+    (tabUrl.includes('linkedin.com/feed') || tabUrl.includes('linkedin.com/in'))
```

Comment:
Checks to see if the user is currently on the LinkedIn feed or an individual profile page.